tls/s2n_config.c

```diff
@@ -501,6 +506,35 @@ int s2n_config_add_cert_chain_and_key_to_store(struct s2n_config *config, struct
     return S2N_SUCCESS;
 }
 
+/* Deprecated. Superseded by s2n_config_add_cert_chain_and_key_to_store */
```

Comment:
Do we want to put the deprecated attribute on the public API so applications know about it?